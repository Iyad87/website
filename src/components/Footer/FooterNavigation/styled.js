import styled from 'styled-components';

import DefaultLink from 'components/Link';
import { footerText } from 'styles/colours';
import { spacing } from 'styles/spacing';

export const Category = styled.ul`
  list-style-type: none;
  margin: 0;
  padding: 0;
  width: 25%;
`;

export const Item = styled.li`
  color: ${props => (props.header ? 'FFFFFF' : footerText)};
  margin-bottom: ${spacing()};
  margin-top: 0;
  font-size: 1.125rem;
  font-style: normal;
  font-weight: ${props => (props.header ? 500 : 400)};
`;

export const Nav = styled.nav`
  display: flex;
  margin: ${spacing(5)} 0;
`;

export const Link = styled(DefaultLink)`
  color: ${footerText};
  font-weight: 400;
`;
