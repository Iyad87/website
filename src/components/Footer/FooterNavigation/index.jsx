import React from 'react';

import { Nav, Category, Item, Link } from './styled';

const Navigation = () => (
  <Nav>
    <Category>
      <Item title>What we do</Item>
      <Item>
        <Link href="/services">Services</Link>
      </Item>
      <Item>
        <Link href="/our-approach">Our Approach</Link>
      </Item>
      <Item>
        <Link href="#how-we-do-it">How we do it</Link>
      </Item>
    </Category>
    <Category>
      <Item title>Our work</Item>
      <Item>
        <Link href="/case-studies">Case Studies</Link>
      </Item>
      <Item>
        <Link href="/industries">Industries</Link>
      </Item>
    </Category>
    <Category>
      <Item title>Who we are</Item>
      <Item>
        <Link href="/about-us">About Us</Link>
      </Item>
      <Item>
        <Link href="/join-us">Join Us</Link>
      </Item>
    </Category>
    <Category>
      <Item title>Our Community</Item>
      <Item>
        <Link href="/talks">Talks</Link>
      </Item>
      <Item>
        <Link href="/events">Events</Link>
      </Item>
      <Item>
        <Link href="/blog">Blog</Link>
      </Item>
    </Category>
  </Nav>
);

export default Navigation;
