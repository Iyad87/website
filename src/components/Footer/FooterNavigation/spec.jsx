import React from 'react';
import { shallow } from 'enzyme';
import FooterNavigation from './index';

test('components/FooterNavigation snapshot', () => {
  const wrapper = shallow(<FooterNavigation />);

  expect(wrapper).toMatchSnapshot();
});
