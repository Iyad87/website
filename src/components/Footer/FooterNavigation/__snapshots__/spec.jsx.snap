// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/FooterNavigation snapshot 1`] = `
<styled.nav>
  <styled.ul>
    <styled.li
      header={true}
    >
      What we do
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/services"
      >
        Services
      </Styled(Link)>
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/our-approach"
      >
        Our Approach
      </Styled(Link)>
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="#how-we-do-it"
      >
        How we do it
      </Styled(Link)>
    </styled.li>
  </styled.ul>
  <styled.ul>
    <styled.li
      header={true}
    >
      Our work
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/case-studies"
      >
        Case Studies
      </Styled(Link)>
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/industries"
      >
        Industries
      </Styled(Link)>
    </styled.li>
  </styled.ul>
  <styled.ul>
    <styled.li
      header={true}
    >
      Who we are
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/about-us"
      >
        About Us
      </Styled(Link)>
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/join-us"
      >
        Join Us
      </Styled(Link)>
    </styled.li>
  </styled.ul>
  <styled.ul>
    <styled.li
      header={true}
    >
      Our Community
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/talks"
      >
        Talks
      </Styled(Link)>
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/events"
      >
        Events
      </Styled(Link)>
    </styled.li>
    <styled.li>
      <Styled(Link)
        href="/blog"
      >
        Blog
      </Styled(Link)>
    </styled.li>
  </styled.ul>
</styled.nav>
`;
